src/main/java/org/jenkinsci/plugins/reverse_proxy_auth/ReverseProxySecurityRealm.java

```diff
@@ -23,61 +23,23 @@
  */
 package org.jenkinsci.plugins.reverse_proxy_auth;
 
-import static hudson.Util.fixEmpty;
-import static hudson.Util.fixEmptyAndTrim;
-import static hudson.Util.fixNull;
-
 import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
 import groovy.lang.Binding;
 import hudson.Extension;
 import hudson.model.Descriptor;
 import hudson.model.User;
-import hudson.security.*;
+import hudson.security.ChainedServletFilter;
+import hudson.security.GroupDetails;
+import hudson.security.LDAPSecurityRealm;
+import hudson.security.SecurityRealm;
+import hudson.security.UserMayOrMayNotExistException;
 import hudson.tasks.Mailer;
 import hudson.tasks.Mailer.UserProperty;
 import hudson.util.FormValidation;
 import hudson.util.Scrambler;
 import hudson.util.spring.BeanBuilder;
-
-import java.io.File;
```

Comment:
Maybe better to keep original simple lexicographic sort?

Reply:
A lot better now :) thanks for the catch